colossus/src/main/scala/colossus/service/RequestHandler.scala

```diff
@@ -58,13 +59,14 @@ abstract class GenRequestHandler[P <: Protocol](val serverContext: ServerContext
   }
 
   def handleRequest(request: Request): Callback[Response] = fullHandler(request)
-  private lazy val errorHandler: ErrorHandler[P]          = onError orElse unhandledError
+
+  private lazy val errorHandler: ErrorHandler[P] = onError orElse unhandledError
 
   def handleFailure(error: ProcessingFailure[Request]): Response = errorHandler(error)
 
   def tagDecorator: TagDecorator[P] = TagDecorator.default[P]
-  def requestLogFormat: Option[RequestFormatter[Request]] =
-    Some(new ConfigurableRequestFormatter[Request](config.errorConfig))
+
+  def requestLogFormat: RequestFormatter[Request] = new StandardRequestFormatter[Request]
```

Comment:
may be out of scope of this PR .. `RequestFormatter` is a bit ambiguous - possibly refactor the name to `RequestExceptionLogger`?

Reply:
`RequestExceptionFormatter` sounds good